test/e2e/pages/vl-map-draw-actions.page.js

```diff
@@ -1,23 +1,24 @@
 const VlMapDrawPointAction = require('../components/vl-map-draw-point-action');
 const VlMapDrawLineAction = require('../components/vl-map-draw-line-action');
 const VlMapDrawPolygonAction = require('../components/vl-map-draw-polygon-action');
-const VlMap = require('../components/vl-map');
-const {Page, Config} = require('vl-ui-core').Test;
+const {Config} = require('vl-ui-core').Test;
+const VlMapPage = require('./vl-map.page');
 
-class VlMapDrawActionsPage extends Page {
+class VlMapDrawActionsPage extends VlMapPage {
   async getMapWithDrawPointAction() {
-    return new VlMap(this.driver, '#map-with-draw-point-action');
+    return this._getMap('#map-with-draw-point-action');
   }
+
   async getMapWithDrawLineAction() {
-    return new VlMap(this.driver, '#map-with-draw-line-action');
+    return this._getMap('#map-with-draw-line-action');
   }
 
   async getMapWithDrawLineAction() {
-    return new VlMap(this.driver, '#map-with-draw-line-action');
+    return this._getMap('#map-with-draw-line-action');
```

Comment:
heb het in mijn PR al aangepast maar we hebben in deze page twee maal de methode getMapWithDrawLineAction()